File: Chapter-3/Java/Problem3.java
import java.util.Scanner;

public class Problem3 {
    public static void main(String[] args) {
        int a, b, c;
        Scanner sc = new Scanner(System.in);
        a = sc.nextInt();
        b = sc.nextInt();
        c = sc.nextInt();
        if (a + b + c == 180) {
            System.out.println("Valid Triangle");
        } else {
            System.out.println("Invalid Triangle");
        }
        sc.close();
    }
}

File: Chapter-3/C programming/Problem3.c
#include <stdio.h>
#include <math.h>

int main(){
    int a,b,c;
    scanf("%d %d %d",&a,&b,&c);
    if(a+b+c == 180){
        printf("Valid Triangle");
    }
    else{
        printf("Invalid Triangle");
    }
}